src/api.v2/controllers/experimentController.js

```diff
@@ -129,11 +137,41 @@ const downloadData = async (req, res) => {
   logger.log(`Providing download link for download ${downloadType} for experiment ${experimentId}`);
 
   const downloadLink = await new Experiment().getDownloadLink(experimentId, downloadType);
+
+  logger.log(`Finished providing download link for download ${downloadType} for experiment ${experimentId}`);
   res.json(downloadLink);
 };
 
+const cloneExperiment = async (req, res) => {
+  const {
+    params: { experimentId: fromExperimentId, toExperimentId },
+  } = req;
+
+  logger.log(`Cloning experiment ${fromExperimentId} into ${toExperimentId}`);
+
+  // toExperiment might have metadata tracks,
+  // we want to start from an empty one so clear them out
+  await new MetadataTrack().delete({ experiment_id: toExperimentId });
```

Comment:
it exists because in the ticket it says we should clone an example experiment into the empty experiment that causes us to show the example experiments suggestion
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/8465903/175060129-936dc096-49ae-4390-be65-6b04760d2625.png">